tmp/rbac.yml

```diff
@@ -0,0 +1,362 @@
+apiVersion: cloud.redhat.com/v1alpha1
```

Comment:
Curious what this tmp file is used for?

Reply:
This is the `ClowdApp` that would be used to deploy RBAC

Reply:
👍

Reply:
There are two other ClowdApp yaml docs in this PR.  Can you explain what distinguishes one from another?